templates/web/zurich/admin/report_edit.html

```diff
@@ -120,106 +105,131 @@
-        <option value="[% cat %]">[% cat %]</option>
-      [% END %]
-    </select></li>
 
-<li class="assignation">
-<label for="body_external">[% loc('Assign to external body:') %]</label>
-    <select name="body_external" id="body_external">
-        <option value="">--</option>
-    [% FOR body IN bodies %]
-        [% NEXT IF body.parent OR body.bodies OR body.deleted %]
-        <option value="[% body.id %]"[% IF body.id == problem.bodies_str %] selected[% END %]>[% body.name %]</option>
-    [% END %]
+[% IF problem.state == 'confirmed' %]
+    <div id="assignation__category">
+    [% loc('Assign to different category:') %]</label>
+    <select name="category" id="category">
+            <option value="">--</option>
+        [% FOREACH cat IN categories %]
+            <option value="[% cat %]">[% cat %]</option>
+        [% END %]
     </select>
-    <br>
-    <input type="checkbox" name="third_personal" id="third_personal" value="1"[% ' checked' IF problem.extra.third_personal %]>
-    <label for="third_personal" class="inline">[% loc('Include reporter personal details') %]</label>
-
-</ul>
-
-<script type="text/javascript">
-$(function(){
-    $('.assignation select').change(function(){
-        if (this.value == "") {
-            $('.assignation').css('color', '#000');
-        } else {
-            var a = $(this).closest('li').css('color', '#000');
-            $('.assignation select').not(this).val("");
-            $('.assignation').not(a).css('color', '#999');
-        }
-    });
-});
-</script>
-
-[% ELSIF problem.state == 'planned' %]
+    </div>
+[% END %]
+</li>
 
-<ul class="no-bullets">
 [% list = FOR body IN bodies %]
     [%- NEXT UNLESS body.parent.id == c.user.from_body.id %]
-    <option value="[% body.id %]">[% body.name %]</option>
+    <option value="[% body.id %]"[% IF body.id == problem.bodies_str %] selected[% END %]>[% body.name %]</option>
 [% END %]
   [% IF admin_type != 'super' AND list %]
-    <li class="assignation">
-    <label for="body_subdivision">[% loc('Assign to subdivision:') %]</label>
+    <li class="assignation" id="assignation__subdivision">
+        <label for="body_subdivision">[% loc('Assign to subdivision:') %]</label>
         <select name="body_subdivision" id="body_subdivision">
-            <option value="" selected>--</option>
+            <option value="">--</option>
             [% list %]
         </select>
     </li>
   [% END %]
 
-    <li><label for="status_update">[% loc('Public response:') %]</label>
-    [% INCLUDE 'admin/response_templates_select.html' for='status_update' %]
-    <textarea name='status_update' id='status_update' cols=60 rows=5>
-      [%- IF problem.extra.public_response -%]
-        [%- problem.extra.public_response | html -%]
-      [%- ELSE -%]
-
-Freundliche Grüsse
+[% END %]
 
-Ihre Stadt Zürich
-    [%- END %]</textarea>
+[%# 3rd party messages sent for Extern/Wunsch states %]
+[% FOR s IN ['closed','investigating'] %]
+    [% IF problem.state == s %]
```

Comment:
`problem.state` can't be both `closed` and `investigating`, and the `s` value doesn't seem to be used within this `FOR` block, so this can probably be simplified to `IF problem.state == 'closed' OR problem.state == 'investigating'`.